core/cfg/flags.go

```diff
@@ -159,7 +162,7 @@ MISC OPTIONS:
 
 		cli.StringFlag{
 			Name:  "endpoint",
-			Value: "https://storage.yandexcloud.net",
+			Value: "",
```

Comment:
How about making tigris endpoint the default?

Reply:
It is default. Set by build system in goreleaser and Makefile.